aggsender/block_notifier_polling_test.go

```diff
@@ -45,9 +46,11 @@ func TestBlockNotifierPollingStep(t *testing.T) {
 	tests := []struct {
 		name                      string
 		previousStatus            *blockNotifierPollingInternalStatus
+		cfg                       *ConfigBlockNotifierPolling
 		HeaderByNumberError       bool
 		HeaderByNumberErrorNumber uint64
```

Comment:
Why are these fields public?


```suggestion
		headerByNumberError       bool
		headerByNumberErrorNumber uint64
```

Reply:
Fix: https://github.com/agglayer/aggkit/pull/617/commits/43247a30e199ed5b7e5dc715789ec4837a04c980